dev/ProgressRing/ProgressRing.xaml

```diff
@@ -20,8 +20,24 @@
                     <Grid x:Name="LayoutRoot" Background="Transparent">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Active" />
-                                <VisualState x:Name="Inactive" />
+                                <VisualState x:Name="Active">
+                                    <Storyboard>
+                                        <DoubleAnimation
+                                            Storyboard.TargetName="IndeterminateAnimatedVisualPlayer"
+                                            Storyboard.TargetProperty="Opacity"
+                                            To="1"
+                                            Duration="0:0:0.15" />
+                                    </Storyboard>
+                                </VisualState>
+                                <VisualState x:Name="Inactive">
+                                    <Storyboard>
```

Comment:
I think just this should be enough. dont have to set it to 1 in Active. VSM should take care of that.

Reply:
It is necessary for the FadeIn transition back to Opacity to 1. However, the debate can also be whether there should be a transition animation or not?

Reply:
Ah i see.

---
In reply to: [406964416](https://github.com/microsoft/microsoft-ui-xaml/pull/2268#discussion_r406964416) [](ancestors = 406964416)